src/api/dataModels.js

```diff
@@ -0,0 +1,36 @@
+import {
+  DATA_MODELS,
+  DATA_MODELS_P1,
+  DATA_MODELS_P2,
+  DATA_MODELS_P3,
+  ERROR,
+} from 'test/mocks/dataModels';
+import throttle from 'util/throttle';
+
+
+export const getDataModels = type => (
+  new Promise((resolve, reject) => {
+    if (type.errors) {
+      throttle(() => resolve(DATA_MODELS.payload));
+    } else {
+      reject(ERROR);
```

Comment:
Use resolve, as fetch only rejects if there's an exception, and not when the server returns an error HTTP status